### PR TITLE
Q4 additions

### PR DESCRIPTION
**Pull request by dziban303**
## Describe your changes

Added MQ-4C Tritons and NATO RQ-4D Phoenixes. Corrected N1DC category to match others in that category (and fix grammar).

Sources include Mictronics and VRS databases; verified reg numbers with publicly available imagery

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Comment by Phaeton** [comment]
On N1DC, it was originally submitted the way you are changing it to, see #72, but the “‘“ and “?” were removed in #76. Not sure if those characters are problematic or not.

**Comment by dziban303** [comment]
I looked at the existing tag frequencies and there were dozens with the
space-question mark, and only one without. I was just trying to get it in
line with the others.

Of course, the other Q-4s are in different categories, some in Oxcart, some
in UAV; it's kind of a mess.

Honestly, many of these tags and categories need to be seriously revamped,
as in many cases the tag is an inside joke few people would get... Like the
`tag tam` or whatever for firefighting aircraft. I'd be surprised if ten
users would be familiar with the Welsh-language children's cartoon.

A question to ask is, does anyone care about these three tag fields or the
category field? Couldn't we throw out ¾s of them in favor of one?

File: plane-alert-civ.csv [hunk 1]
```diff
@@ -3326,7 +3326,7 @@ C0010A,C-FAKF,Private,AutoGyro Europe Cavalon,CLON,Civ,Little Nellie,Gyrocopter,
 A77B6F,N581UP,United Parcel Service,Boeing 747,B744,Civ,You Came Here In That Thing,Flying Rust Bucket,Nope Mobile,You came here in that thing ?,https://www.ups.com/gb/en/Home.page
 A3026F,N293UP,United Parcel Service,MD-11,MD11,Civ,You Came Here In That Thing,Flying Rust Bucket,Nope Mobile,You came here in that thing ?,https://www.ups.com/gb/en/Home.page
 406E4D,G-CIRH,Willy Rose Technology,Magni Gyro M-16 Tandem Trainer,MM16,Civ,Little Nellie,Gyrocopter,Nope Mobile,You came here in that thing ?,http://www.magnigyro.it/en/products/m24-orion
-A00050,N1DC,Jerry Jones,Gulfstream G-V,GLF5,Civ,NFL,The Other Football,Americas Team,Dont you know who I am,https://en.wikipedia.org/wiki/Dallas_Cowboys
+A00050,N1DC,Jerry Jones,Gulfstream G-V,GLF5,Civ,NFL,The Other Football,America's Team,Don't you know who I am ?,https://en.wikipedia.org/wiki/Dallas_Cowboys
```
Comment (Phaeton) [inline plane-alert-civ.csv:3329]:
```suggestion
A00050,N1DC,Jerry Jones,Gulfstream G-V,GLF5,Civ,NFL,The Other Football,America's Team,Don't you know who I am?,https://en.wikipedia.org/wiki/Dallas_Cowboys
```

**Comment by dziban303** [comment]
But the space is in all of the entries of that category. It's also in nearly 100 entries of the category 'You came here in that thing ?'

Output from `xsv frequency -l 0 -s Category plane-alert-db.csv`:

```
Category,You came here in that thing ?,98
...
Category,Don't you know who I am ?,17
```

So if the space was removed from one, it should be removed from them all.

**Comment by dziban303** [comment]
All instances of `blah blah ?` now changed to `blah blah?`

**Pull request closed without merging**